############### Distance estimation parameters ###############
# distance estimation from object size parameters
max_dist: 10.0
max_dist_diff: 4.0
# depthmap parameters
min_depth: 200 # millimetres
max_depth: 12000 # millimetres
distance_min_valid_pixels_ratio: 0.05
object:
  physical_height: 1.9 # metres

debug_image_source: 0

 ############### Other parameters ###############
# xy_covariance_coeff: 0.001
# z_covariance_coeff: 0.005
loop_rate: 100 # how often the detection loop should run
cov_coeffs:
  xy:
    no_estimate: 2.5
    blob_size:   0.05
    depthmap:    0.04
    both:        0.03
  z:
    no_estimate: 666.0
    blob_size:   0.5
    depthmap:    0.3
    both:        0.1
